deepchem/trans/tests/test_transformers.py

```diff
@@ -482,3 +485,42 @@ def test_featurization_transformer(self):
 
     self.assertEqual(new_train.y.shape, train.y.shape)
     self.assertEqual(new_train.X.shape[-1], fp_size)
+
+  def test_data_transformers(self):
+    mnist = input_data.read_data_sets("MNIST_data/", one_hot=True)
+    # extracting validation set of MNIST for testing the DataTransforms
+    valid = dc.data.NumpyDataset(mnist.validation.images,
+                                 mnist.validation.labels)
+    # extract only the images (no need of the labels)
+    data = (valid.X)[0]
+    # reshaping the vector to image
+    data = np.reshape(data, (28, 28))
+
+    dt = DataTransforms(data)
```

Comment:
It is better to split the test by function instead of a large single test.  You can put loading the MNIST dataset into the class constructor to only have to load it once.

Reply:
I did that since all the data transforms were written in a single class. I will change it.

Reply:
This is still one test instead of one per function